src/calculateRentalCost.js

```diff
@@ -5,6 +5,13 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else {
+    return days * 40;
+  }
```

Comment:
You can simplify the code by removing the `else` statement. If the condition in the `if` is met and the function returns, the following code will not be executed anyway, making the `else` unnecessary and reducing the level of indentation.